packages/components/src/internal/components/chart/ChartBuilderModal.tsx

```diff
@@ -698,9 +647,12 @@ export const ChartBuilderModal: FC<ChartBuilderModalProps> = memo(({ actions, mo
                     fieldValues_['scales'] = { value: { ...chartConfig.scales } };
                 }
 
-                // handle bar chart aggregate method
+                // handle bar chart aggregate method and error bars
```

Comment:
I know you're only adding a little bit of logic to this useEffect, but it should not be a useEffect. Instead this should be a useMemo. Nothing in this useEffect is loading anything.